api/tests/helpers/database.ts

```diff
@@ -5,6 +5,7 @@ export function cleanDatabase() {
   const clean = async () => {
     await prisma.coreUnit.deleteMany();
     await prisma.user.deleteMany();
+    await prisma.session.deleteMany();
```

Comment:
i've tried these both **briefly** before submitting this line.

- both break tests.
  - with strange errors
  - not a surprise for direct query since it's for psql

- the github solutions (the linked one and those below) rely on specific prisma versions apparently (prisma obj layout) and on top of this there's a good point at https://github.com/prisma/docs/issues/451#issuecomment-1354500062

Given that suggested programmatic solutions do not work out cleanly, i would ask the following:

Should i write raw sqlite process here or should i just write manual command that deletes and recreates the db file? latter might be less efficient in runtime, but is simpler. If we intend to migrate to psql eventually, it's a quick and ez way forward.